krm-functions/nfdeploy-fn/mutator/helper.go

```diff
@@ -0,0 +1,85 @@
+package mutator
+
+import (
+	nephiodeployv1alpha1 "github.com/nephio-project/api/nf_deployments/v1alpha1"
+	kptcondsdk "github.com/nephio-project/nephio/krm-functions/lib/condkptsdk"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+type NfDeployFn struct {
+	sdk                           kptcondsdk.KptCondSDK
+	site                          string
+	capacityMaxUpLinkThroughPut   resource.Quantity
+	capacityMaxDownLinkThroughPut resource.Quantity
+	networkInstance               map[string]nephiodeployv1alpha1.NetworkInstance
+	interfaceConfigsMap           map[string]nephiodeployv1alpha1.InterfaceConfig
+}
+
+func NewMutator() NfDeployFn {
+	return NfDeployFn{
+		interfaceConfigsMap: make(map[string]nephiodeployv1alpha1.InterfaceConfig),
+		networkInstance:     make(map[string]nephiodeployv1alpha1.NetworkInstance),
+	}
+}
+
+func (h *NfDeployFn) SetInterfaceConfig(interfaceConfig nephiodeployv1alpha1.InterfaceConfig, networkInstanceName string) {
```

Comment:
Are the Set*** methods needed if the fields are manipulated locally? I can understand if you're using it for test/mock purposes.